nav_msgs/Path plan
---
string response

